src/webui/webapplication.cpp

```diff
@@ -533,7 +533,7 @@ void WebApplication::sessionInitialize()
 {
     Q_ASSERT(!m_currentSession);
 
-    const QString sessionId {parseCookie(m_request.headers.value(QLatin1String("cookie"))).value(C_SID)};
+    const QString sessionId {parseCookie(m_request.headers.value(QLatin1String("cookie"))).value(QString::fromLatin1(C_SID))};
```

Comment:
```suggestion
    const QString sessionId {parseCookie(m_request.headers.value(u"cookie"_qs)).value(QString::fromLatin1(C_SID))};
```

And why didn't you turn C_SID into QString?

Reply:
`C_SID` is mainly provided for constructing `QNetworkCookie` which takes `QByteArray` type.